build/build-lightrider-bundles.js

```diff
@@ -5,21 +5,16 @@
  */
 'use strict';
 
-const browserify = require('browserify');
 const rollup = require('rollup');
 const rollupPlugins = require('./rollup-plugins.js');
 const fs = require('fs');
 const path = require('path');
 const bundleBuilder = require('./build-bundle.js');
 const {LH_ROOT} = require('../root.js');
-const inlineFs = require('./plugins/browserify-inline-fs.js');
```

Comment:
Can we del this now?

Reply:
Eh, it's harmless left alone. I'd like to defer to brendan on this file. He (or perhaps just I) would like to extract this into its own published package eventually